CustomTabBar/TabButton.swift

```diff
@@ -156,7 +164,21 @@ private extension TabButton {
             titleLabel.font = UIFont.systemFont(ofSize: 16)
         }
     }
-    
+
+    func updateBadge() {
+        guard let value = dataObject.badgeValue else { badgeLabel.removeFromSuperview(); return }
+        badgeLabel.text = value
+        badgeLabel.textAlignment = .center
+        badgeLabel.backgroundColor = .red
+        if let badgeFont = badgeFont {
+            badgeLabel.font = badgeFont
+        } else {
+            badgeLabel.font = UIFont.boldSystemFont(ofSize: 12)
+        }
+        badgeLabel.textColor = .white
```

Comment:
It'd be good to have this as a parameter too. Assuming that we have the text color of the tabs as a parameter. If not, let's not worry about this either.